libs/astx/src/astx/comprehensions.py

```diff
@@ -146,3 +146,37 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
             **generators,
         }
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class GeneratorExpr(Comprehension):
+    """AST class for generator expressions."""
+
+    element: Expr
+
+    def __init__(
+        self,
+        element: Expr,
+        generators: Iterable[ComprehensionClause]
+        | ASTNodes[ComprehensionClause],
+        loc: SourceLocation = NO_SOURCE_LOCATION,
+        parent: Optional[ASTNodes] = None,
+    ) -> None:
+        """Initialize the GeneratorExpr instance."""
+        super().__init__(generators=generators, loc=loc, parent=parent)
+        self.element = element
+        self.kind = ASTKind.GeneratorExprKind
+
+    def __str__(self) -> str:
```

Comment:
It is not actually necessary, because in theory, it will get the name of the class automatically.

Reply:
but it is not a big deal, no worries to keep it here.